src/metabase/models/pulse_channel.clj

```diff
@@ -128,15 +130,15 @@
   (concat
    (for [email emails]
      {:email email})
-   (for [user (mdb.query/query
-               {:select    [:u.id :u.email :u.first_name :u.last_name]
-                :from      [[:core_user :u]]
-                :left-join [[:pulse_channel_recipient :pcr] [:= :u.id :pcr.user_id]]
-                :where     [:and
-                            [:= :pcr.pulse_channel_id pulse-channel-id]
-                            [:= :u.is_active true]]
-                :order-by [[:u.id :asc]]})]
-     (user/add-common-name user))))
+   (t2/select
+    [User :id :email :first_name :last_name]
+    {:select    [:u.id :u.email :u.first_name :u.last_name]
+     :from      [[:core_user :u]]
+     :left-join [[:pulse_channel_recipient :pcr] [:= :u.id :pcr.user_id]]
+     :where     [:and
+                 [:= :pcr.pulse_channel_id pulse-channel-id]
+                 [:= :u.is_active true]]
+     :order-by [[:u.id :asc]]})))
```

Comment:
Yet another place where we don't manually have to do the `post-select` stuff ourselves